src/styles/partials/components/_site-status.scss

```diff
@@ -0,0 +1,9 @@
+.dg_status-text{
+    color: $gray-light;
+    font-size: $h2-font-size;
+    text-transform: uppercase;
+    font-family: $default-heading-font;
+    font-weight: $font-medium;
+    letter-spacing: $default-padding;
```

Comment:
sorry missed this one, @theilman7 suggested some variables for this, but i dont have a lot of experience with letter-spacing and line-height. @theilman7 what do you think?

Reply:
we can come back to this, isn't a huge deal right now.